examples/aws-sagemaker/config.yaml

```diff
@@ -1,7 +1,10 @@
-models:
+num_cores: 1
+num_workers: 1
+integration: sagemaker
```

Comment:
btw @bfineran, do we even need this if we can just directly set the route below to `/invocations`?